### ISSUE TITLE
not able to find cordova.js

### ISSUE THREAD

**Issue by arorasahil1234** [evothings/cordova-ble#154]
<script src="cordova.js"></script>

where is this file buddy ???

**Comment by kinesias** [comment]
Do you even know how Cordova works? Cordova.js is being  generated when you do „cordova build“

**Comment by arorasahil1234** [comment]
I agree , I am new to this framework , could you please help me how can i set the region for beacon based on id ,major or minor ??
